java-squid/src/main/java/org/sonar/java/se/ConstraintManager.java

```diff
@@ -130,21 +133,22 @@ private static Tree skipTrivial(Tree expression) {
 
   }
 
-  public boolean isNull(ProgramState ps, SymbolicValue val){
+  public boolean isNull(ProgramState ps, SymbolicValue val) {
     return NullConstraint.NULL.equals(ps.constraints.get(val));
   }
 
   public Pair<ProgramState, ProgramState> assumeDual(ProgramState programState, Tree condition) {
-    //FIXME condition value should be evaluated to determine if it is worth exploring this branch. This should probably be done in a dedicated checker.
+    // FIXME condition value should be evaluated to determine if it is worth exploring this branch. This should probably be done in a
+    // dedicated checker.
     condition = skipTrivial(condition);
     switch (condition.kind()) {
       case INSTANCE_OF: {
         InstanceOfTree instanceOfTree = (InstanceOfTree) condition;
         SymbolicValue exprValue = eval(programState, instanceOfTree.expression());
-        if(isNull(programState, exprValue)) {
+        if (isNull(programState, exprValue)) {
           return new Pair<>(programState, null);
         }
-        //if instanceof is true then we know for sure that expression is not null.
+        // if instanceof is true then we know for sure that expression is not null.
         return new Pair<>(programState, setConstraint(programState, exprValue, NullConstraint.NOT_NULL));
       }
       case EQUAL_TO: {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this nested code block into a method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1199)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 15 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1151)